site/pages/about-us/timeline-slice/index.js

```diff
@@ -87,6 +90,9 @@ class TimelineSlice extends React.Component<*, TimelineSliceState> {
               <div className={styles.imageWrapper}>
                 <img src={Image2016} alt="year 2016" className={styles.image} />
               </div>
+              <div className={styles.imageWrapper}>
+                <img src={Image2017} alt="year 2017" className={styles.image} />
+              </div>
               <Six />
```

Comment:
Think we are missing `Seven` here and the order of components needs to be different too?